tests/users/test_api_change_password.py

```diff
@@ -0,0 +1,114 @@
+from tests.test_utils import get_test_request_header
+
+
+# Testing /PUT/ Change User's Password
+
+class TestUserChangePasswordApi(BaseTestCase):
+
+    def setUp(self):
+        super(TestUserChangePasswordApi, self).setUp()
+        self.first_user = UserModel(
+            password=user1['password'],
+            name="User1",
+            email="user1@email.com",
+            username="user_not_admin",
+            terms_and_conditions_checked=True
+        )
+        self.first_user.is_email_verified = True
+
+        db.session.add(self.first_user)
+        db.session.commit()
+
+        self.current_password = user1['password']
+        self.auth_header = get_test_request_header(self.first_user.id)
+
+    def test_change_password_to_correct_new_one(self):
+        with self.client:
+            new_password = '123new_password'
+            expected_response = PASSWORD_SUCCESSFULLY_UPDATED
+            response = self.client.put('/user/change_password',
+                                       json={'current_password': self.current_password,
+                                             'new_password': new_password},
+                                       follow_redirects=True,
+                                       headers=self.auth_header,
+                                       )
+            self.assertEqual(201, response.status_code)
+            self.assertEqual(expected_response, json.loads(response.data))
+
+    def test_change_passport_with_authentication_token_missing(self):
+        with self.client:
+            new_password = '123new_password'
+            expected_response = AUTHORISATION_TOKEN_IS_MISSING
+            response = self.client.put('/user/change_password',
+                                       json={'current_password': self.current_password,
+                                             'new_password': new_password},
+                                       follow_redirects=True,
+                                       )
+            self.assertEqual(401, response.status_code)
+            self.assertEqual(expected_response, json.loads(response.data))
+
+    def test_change_password_to_equal_one(self):
+        with self.client:
+            new_password = self.current_password
+            expected_response = NEW_PASSWORD_IS_EQUAL_TO_CURRENT_PASSWORD
+            response = self.client.put('/user/change_password',
+                                       json={'current_password': self.current_password,
+                                             'new_password': new_password},
+                                       follow_redirects=True,
+                                       headers=self.auth_header,
+                                       )
+            self.assertEqual(400, response.status_code)
+            self.assertEqual(expected_response, json.loads(response.data))
+
+    def test_change_password_to_empty_one(self):
+        with self.client:
+            new_password = ''
```

Comment:
Can you add a message for this? Refer this method `test_user_password_change_request_password_inferior_to_limit`